submissions/benjaminKamena/phpDataase/queryDb.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include('cookie.php');
+
+$results = isset($_COOKIE['customers'])? unserialize($_COOKIE['customers']) : $queryDatabase->getAllCustomers();
+
+try{
+	if(count($results) > 0){
+		echo "<table>";
+		   echo "<tr>";
+		     echo"<th>SN</th>";
+		     echo"<th>fullName</th>";
+		     echo"<th>EmailAddress</th>";
+		     echo"<th>CreatedAt</th>";
+		     echo "<th>Actions</th>";
+		   echo"</tr>";
+    foreach($results as $row){
+		echo "<tr>";
+           echo "<td> .$row[SN].</td>";
```

Comment:
What is the reason for the `.`

Reply:
modified the code

Reply:
To refer to the exit task 2

Reply:
Are there columns  in your table called `SN` and `Actions`

Reply:
That is wrong. It should be that of `columns` you have on a table

Reply:
This is your `customers` table 
![image](https://user-images.githubusercontent.com/43061141/105680473-160c5100-5ef0-11eb-8c13-4797082db41c.png) And it is not there.

Reply:
Also, for your email, it is `customer_email_address` not `EmailAddress`. And there is no column called `fullName`. It is `customer_first_name` and `customer_last_name`

Reply:
Kindly make all the necessary corrections. I am working with the existing eCommerce database you created. Don't create a new one for yourself